lib/util/constants.js

```diff
@@ -23,10 +23,6 @@ export const CREATE_TRIP_PATH = `${TRIPS_PATH}/new`
 export const TERMS_OF_SERVICE_PATH = '/terms-of-service'
 export const TERMS_OF_STORAGE_PATH = '/terms-of-storage'
 
-// Gets the root URL, e.g. https://otp-instance.example.com:8080, computed once for all.
-// TODO: support root URLs that involve paths or subfolders, as in https://otp-ui.example.com/path-to-ui/
```

Comment:
I don't think we fixed this TODO did we?

Reply:
We have not. Noted in #652.